docs/docs/user-guide/run-cli-in-docker-container.md

```diff
@@ -0,0 +1,101 @@
+# Run CLI for Microsoft 365 in a Docker Container
```

Comment:
Shall we add a section explaining that the auth information is not persisted in the image and after you exit the container, you will need to login the next time you start a new one?

Reply:
Good observation, yes that's a good idea,  added 👍